component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/tiqr/TiqrAuthenticator.java

```diff
@@ -373,7 +373,10 @@ private String sendRESTCall(String url, String urlParameters, String formParamet
             }
             return TiqrConstants.FAILED + e.getMessage();
         } finally {
-            connection.disconnect();
+            if (connection != null) {
```

Comment:
HttpURLConnection is not implements from AutoClosable interface, so we can't use it with try with resource.